packages/connect-ui/src/components/Layout.tsx

```diff
@@ -45,7 +46,9 @@ export const Layout: React.FC = () => {
     }
 
     return (
-        <div className="absolute h-screen w-screen overflow-hidden flex flex-col justify-center items-center p-14 bg-subtle/80">
+        <div
+            className={`absolute h-screen w-screen overflow-hidden flex flex-col justify-center items-center p-14 ${isAuthLink ? (isDarkTheme ? 'bg-black' : 'bg-gray-100') : 'bg-subtle/80'}`}
```

Comment:
```suggestion
            className={cn('absolute h-screen w-screen overflow-hidden flex flex-col justify-center items-center p-14 bg-subtle/80', isAuthLink && 'bg-gray-100 dark:bg-black')}
```
> Don't directly apply this suggestion from gh. `cn` has to be imported and `isDarkMode` deleted

Ideally we ask Gab for colors here. I think the background gradient from the webapp would look nice in dark mode. Here's my test:
<img width="1511" height="804" alt="Image" src="https://github.com/user-attachments/assets/a75127a3-7651-4256-8717-d3fd63edf25f" />

Reply:
Yeah, known issue with the design. Will iterate on it in the future